src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/tests/__snapshots__/FieldBlocks.test.js.snap

```diff
@@ -46,9 +46,6 @@ exports[`Render block with schema 1`] = `
         <header
           class="header"
         >
-          <div
-            class="icons"
-          />
```

Comment:
> are the icons now skipped on initial rendering? Or why is this icons div now away?

Because we have skip the rendering now completely on this line `Object.keys(this.iconsMapping).length === 0` when the iconsMapping is empty. Previously we did return an empty array which did render the icons div but without icons.

Reply:
Ah sorry I did not explain it correctly, it returned empty arrays in an empty array, like `[ [], [] ]`